app/models/email.rb

```diff
@@ -26,35 +26,35 @@ def self.weekly_patrol_rosters(organisation = nil)
           next_roster = user_roster.first
           if user_roster.second.present?
             following_roster = user_roster.second
-            if user.club.is_active && user.club.enable_reminders_email
-              SwapseaMailer.weekly_roster_reminder(user, next_roster, following_roster, subject).deliver
+            if user.club.with_reminder_emails_enabled
+              SwapseaMailer.roster_reminder(user, next_roster, following_roster, subject).deliver
               emails_sent += 1
             else
               Rails.logger.warn "Skipped sending patrol roster email because #{user.club.name} is_active=#{user.club.is_active} and enable_reminders_email=#{user.club.enable_reminders_email}"
             end
-            if user.club.is_active && user.club.enable_reminders_sms
-              SwapseaSms.weekly_roster_reminder(user, next_roster).deliver
+            if user.club.with_reminder_sms_enabled
+              SwapseaSms.roster_reminder(user, next_roster).deliver
               sms_sent += 1
             else
               Rails.logger.warn "Skipped sending patrol roster SMS because #{user.club.name} is_active=#{user.club.is_active} and enable_reminders_sms=#{user.club.enable_reminders_sms}"
             end
           elsif user_roster.first.present?
-            if user.club.is_active && user.club.enable_reminders_email
-              SwapseaMailer.weekly_roster_reminder(user, next_roster, nil, subject).deliver
+            if user.club.with_reminder_emails_enabled
+              SwapseaMailer.roster_reminder(user, next_roster, nil, subject).deliver
               emails_sent += 1
             else
               Rails.logger.warn "Skipped sending patrol roster email because #{user.club.name} is_active=#{user.club.is_active} and enable_reminders_email=#{user.club.enable_reminders_email}"
             end
-            if user.club.is_active && user.club.enable_reminders_sms
-              SwapseaSms.weekly_roster_reminder(user, next_roster).deliver
+            if user.club.with_reminder_sms_enabled
+              SwapseaSms.roster_reminder(user, next_roster).deliver
```

Comment:
Update to ".deliver_now" or ".deliver_later". Deliver later will make it async using ActiveJob.